addons/controllers/clusterbootstrap_controller.go

```diff
@@ -509,6 +517,13 @@ func (r *ClusterBootstrapReconciler) createOrPatchPackageInstallOnRemote(cluster
 				Prereleases: &versions.VersionSelectionSemverPrereleases{},
 			},
 		}
+
+		if remotePkgi.Labels == nil {
```

Comment:
question: `remotePkgi == nill` seems always true ?

Reply:
True, also changed to add label in the block of L499